site/src/pages/WorkspacesPage/WorkspacesPageView.tsx

```diff
@@ -14,15 +14,18 @@ import { FC } from "react"
 import { Link as RouterLink } from "react-router-dom"
 import * as TypesGen from "../../api/typesGenerated"
 import { AvatarData } from "../../components/AvatarData/AvatarData"
+import { EmptyState } from "../../components/EmptyState/EmptyState"
 import { Margins } from "../../components/Margins/Margins"
 import { Stack } from "../../components/Stack/Stack"
+import { TableLoader } from "../../components/TableLoader/TableLoader"
 import { getDisplayStatus } from "../../util/workspace"
 
 dayjs.extend(relativeTime)
 
 export const Language = {
   createButton: "Create workspace",
-  emptyView: "so you can check out your repositories, edit your source code, and build and test your software.",
+  emptyMessage: "Create your first workspace",
+  emptyDescription: "To start edit your source code, and build your awesome software.",
```

Comment:
- Feel free to suggest one if you have one in mind! 
- Hmm... not sure on this one because we only use the CLI in the template because there is no way to create a template from the UI right now.